internal/fdo_test.go

```diff
@@ -0,0 +1,204 @@
+package internal
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"fyne.io/desktop"
+	"fyne.io/fyne"
+	_ "fyne.io/fyne/test"
+	"github.com/magiconair/properties/assert"
+)
+
+var iconTheme = "default_theme"
+var iconSize = 32
+
+type dummyWindow struct {
+}
+
+type dummyWindow1 struct {
```

Comment:
This is a lot of boilerplate structs - why not just create a single one with some state?
It would be much easier to read - you'd put the setup code next to the code testing it.